szn-select--ui.js

```diff
@@ -201,6 +202,46 @@
     if (suggestedHeight !== currentHeight) {
       instance._dropdownContent.style.height = `${suggestedHeight}px`
     }
+
+    const buttonWidth = instance._button.offsetWidth
```

Comment:
Iste, je to zlepšenie, ale stále to predstavuje pre mnohé prípady (vždy keď nie je select fluidne veľký) zbytočné čítanie a zápis do DOMu. Obal to prosím do podmienky na nejaký flag, napr. `if (instance._autoStretchDropdown) { ...` a mirroruj `_autoStretchDropdown` na `<szn-select>.autoStretchDropdown` tak ako to robíme s `dropdownClassName`, prosím.

Reply:
zmenil jsem jen trochu nazev na dropdownAutoStretch, aby stejne jako dropdownClassName zacinal na dropdown...